ui/src/message_popup/scaffolding_card.jsx

```diff
@@ -46,21 +46,21 @@ export default React.createClass({
   getQuestions(selectedIndicatorId){
     var {isSolutionMode} = this.state;
     const questions = (isSolutionMode || selectedIndicatorId === ALL_INDICATORS)
-      ? _.shuffle(withStudents(allQuestions))
```

Comment:
@david-rosales Ping on this, mind re-reviewing this part when you get a chance?  Thanks!